apps/src/applab/designMode.js

```diff
@@ -309,6 +309,38 @@ designMode.updateProperty = function(element, name, value) {
     case 'readonly':
       element.setAttribute('contenteditable', !value);
       break;
+    case 'data-is-default':
+      if (value === true) {
+        //Make this one default
+        $('#designModeViz .screen').attr('data-is-default', false);
+        element.setAttribute('data-is-default', true);
+        $('#designModeViz').prepend(element);
+
+        //Resort elements in the dropdown list
+        var options = $('#screenSelector option');
+        var newScreenText = options.last().text();
+        var defaultScreenId = elementUtils.getId(element);
+        options.sort(function (a, b) {
+          if (a.text === defaultScreenId) {
+            return -1;
+          } else if (b.text === defaultScreenId) {
+            return 1;
+          } else if (a.text === newScreenText) {
+            return 1;
+          } else if (b.text === newScreenText) {
+            return -1;
+          } else {
+            return a.text.localeCompare(b.text);
+          }
+        });
```

Comment:
Ahh, misunderstood what was happening here. Looks like we already have a constant NEW_SCREEN - would it make more sense to use that instead of options.last().text()?